packages/flutter_tools/lib/src/android/android_device.dart

```diff
@@ -800,9 +801,18 @@ Map<String, String> parseAdbDeviceProperties(String str) {
 /// For more information, see https://developer.android.com/studio/command-line/dumpsys.
 @visibleForTesting
 AndroidMemoryInfo parseMeminfoDump(String input) {
+print("=====> ENTER parseMeminfoDump");
   final AndroidMemoryInfo androidMemoryInfo = AndroidMemoryInfo();
-  input
-    .split('\n')
+  
+  final lines = input.split('\n');
```

Comment:
To make the lints happy here you'll need to declare the types

Reply:
Same here and a couple of other places too (forgot to his save).  On a different machine w/o auto-save (turned on now ;-).